symengine/lib/symengine_wrapper.pyx

```diff
@@ -225,7 +225,7 @@ def sympy2symengine(a, raise_error=False):
         for r in a.tolist():
             for e in r:
                 v.append(e)
-        return DenseMatrix(row, col, v)
+        return DenseMatrixBase(row, col, v)
```

Comment:
This should be `MutableDenseMatrix` or `ImmutableDenseMatrix` depending on the sympy type